Legion.FrontEnd/package.json

```diff
@@ -44,7 +44,10 @@
     ]
   },
   "dependencies": {
+    "@mdi/font": "^2.5.94",
     "history": "^4.7.2",
+    "mdi-react": "^4.0.0",
```

Comment:
Makes it easier to use Material Design Icons

Reply:
Hmm nifty!  I'll have to check it out.

Reply:
After briefly looking at it, I'm fine with using it for now.  The library isn't hugely popular, but it is being maintained, so that's good.

Reply:
Yeah. That was my main issue. I wanted something that was easy to use, and currently maintained. There are some issues with using it, but it is very user friendly for newcomers to React and JS.